brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumerTest.java

```diff
@@ -1108,9 +1128,9 @@ private String extractActualHandleReportFromS3Client(S3Event s3Event, Publicatio
     private UriWrapper constructHandleReportFileUri(S3Event s3Event, Publication actualPublication) {
         var timestamp = s3Event.getRecords().get(0).getEventTime().toString(YYYY_MM_DD_HH_FORMAT);
         return UriWrapper.fromUri(HANDLE_REPORTS_PATH)
-                   .addChild(actualPublication.getResourceOwner().getOwner().getValue().split("@")[0])
-                   .addChild(timestamp)
-                   .addChild(actualPublication.getIdentifier().toString());
+                                    .addChild("institution")
```

Comment:
Do we know that there is no two Brage posts in the same archive that points to the same NVA publication?

If yes, then this is fine, if no. We need to change the path to somehow reflect the Brage-handle and not the NVA publication identifier.